app/helpers/record_helper.rb

```diff
@@ -98,6 +98,48 @@ def access_type(metadata)
     access_right.first['description']
   end
 
+  def issued_date(dates)
+    return if dates.blank? || dates&.none? { |date| date['kind'] == 'Issued' }
+
+    # If there is more than one date issued, take the first one.
```

Comment:
I'm not sure this is the behavior we want. I feel we should display multiple dates in this situation, either as comma separate or as a bulleted list.